### DISCUSSION TITLE
Error creating app (cloud sandbox)

### DISCUSSION THREAD

**Discussion by nathanielconroy** [Budibase/budibase#5774] [Help]
I'm new to budibase and attempting to give it a test run in the cloud sandbox. However, I receive an "Error creating app" message every time I try to create a new app, either from scratch or from a template. I'm sure I could troubleshoot this better with a self-hosted app, but I'd like to get a better sense of budibase's capabilities before investing the time in self-hosting. 

My tenant id is nathantestapp

**Comment by melohagan** [comment]
Hey @nathanielconroy 

Thanks for your interest in Budibase! Sorry for the roadblock. 

Would you mind sharing any errors from the network tab? (Fn+F12 on Chrome)

There is a similar issue that is currently being investigated here: https://github.com/Budibase/budibase/issues/5738

**Reply by Rory-Powell** [reply]
The issue linked above is the same route cause for this one @nathanielconroy, some more details in this comment https://github.com/Budibase/budibase/issues/5738#issuecomment-1120892266

The best steps right now would be to create a new account without the word `app` in the tenant id.

**Reply by nathanielconroy** [reply]
Thanks for the work around! That seems to resolve my issue.
